### ISSUE TITLE
update syntax of manifest from "applications" to "browser_specific_settings"

### ISSUE THREAD

**Issue by jraller** [freelawproject/recap#268]

**Comment by jraller** [comment]
Chrome complains about the syntax of the current manifest file, pointing out that "applications" is not part of the spec. Looking at https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_specific_settings it appears that at some point Mozilla used "applications" but has since replaced it with "browser_specific_settings".

**Comment by jraller** [comment]
I found https://wiki.mozilla.org/WebExtensions/Applications which covers the name change.

**Comment by jraller** [comment]
Chrome still complains about browser_specific_settings at this point. Low benefit in changing this at this point. Keep issue open and recheck in future.

**Comment by mlissner** [comment]
Yeah, one of those annoying things of cross-browser dev. Good to know it's still funky. I suppose we could fix this with some kind of build system, but that'd be overkill.
